### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by regadas**

### CODE REVIEW

File: src/main/g8/build.sbt [hunk 1]
```diff
@@ -9,7 +9,7 @@ lazy val commonSettings = Defaults.coreDefaultSettings ++ Seq(
   organization          := "$organization$",
   // Semantic versioning http://semver.org/
   version               := "0.1.0-SNAPSHOT",
-  scalaVersion          := "2.11.12",
```
Comment (ravwojdyla) [inline src/main/g8/build.sbt]: There were some issues with serialization etc in `2.12`, might be safer to stay on `2.11`

File: src/main/g8/project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=1.1.0
+sbt.version=1.1.1
```
Comment (ravwojdyla) [inline src/main/g8/project/build.properties]: + new line

File: src/main/g8/build.sbt (hunk 1 above)
Comment (regadas) [reply src/main/g8/build.sbt]: @ravwojdyla sorry for late reply, hmm yeah, I haven't seen any of those for awhile now, that's why I figure we could push it. But if you wanna keep it let me know.

Comment (ravwojdyla) [reply src/main/g8/build.sbt]: I would prefer staying on `2.11`, and leave it up to people to change if they want.

Comment (regadas) [reply src/main/g8/build.sbt]: @ravwojdyla ok 😄

**Pull request merged**